CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Breaking changes:
   Most browsers define a page as served from secure context if its url has `https` scheme. For local testing `localhost` is treated as a secure context as well.
   Unfortunately, it does not work nicely in FireFox, since it does not treat subdomains of localhost as secure as well, compare to Chrome.
   If you want to test with FireFox you can configure it as described [here](https://github.com/eclipse-theia/theia/pull/6465#issuecomment-556443218).
+  - [bunyan] removed `@theia/bunyan` extension []()
```

Comment:
We could add the following link, that shows the last published version of this extension's source code: https://github.com/eclipse-theia/theia/tree/b92a5673de1e9d1bdc85e6200486b92394200579/packages/bunyan

Reply:
Updated based on your suggestion :)